CONTRIBUTING.md

```diff
@@ -7,10 +7,10 @@ Getting started
 If you want to help but don't know where to start, here are some
 low-risk/isolated tasks:
 
-- [Merge a Vim patch].
```

Comment:
I agree in many cases it's not easy but for some patches it is doable and we 've had several first-time contributions merging vim patches so I wouldn't write it off yet

Reply:
And knowing which patches are doable and which are a rabbit hole into a warren of madness is not easy for new contributors to tell... ;)

We don't have to remove it, but I do believe we should not lead with it (and add the recommendation to ask on Gitter first before trying to drink from the firehose).